helm/kubernetes-cluster-prep/test-helm

```diff
@@ -13,59 +18,53 @@ print_usage() {
   echo "Syntax:"
   echo "    $0 [Options]"
   echo "    Options:"
-  echo "    -h           Show help"
-  echo "    -i           Install the Helm release"
```

Comment:
I'm thinking it's cleaner to assume, as a pre-requisite, that the user has already installed a release, and is now running `helm test ...` to validate what is deployed.

If we were to include the install of a Helm release as part of this script, it adds the complexity to this script that the user would need to provide meaningful Conjur connect info, either as environment variables, or as a custom values.yaml file. Seems cleaner (more composable?) to me to separate these operations.

An alternative could be to add a `-t` or test flag to our Helm install script to have that script to `helm test ...` after install. Since the Helm install script handles the Conjur connect info already, adding the test options would be a small addition.

Reply:
OK, I like the -t option after the install, but that doesn't have to be part of this.